silvestre-ffi/src/lib.rs

```
@@ -1,13 +1,1130 @@
+
+        let rest = json[pos + search.len()..].trim_start();
+        if let Some(after) = rest.strip_prefix(':') {
+            break after.trim_start();
+        }
+        // This occurrence wasn't a key (no colon after it), keep searching.
+        search_from = pos + search.len();
+    };
+
+    // If the value is a quoted string
+    if after_colon.starts_with('"') {
+        let content = &after_colon[1..];
```

Clippy (1.95.0)

[warning] stripping a prefix manually

(warning)